scipy/stats/stats.py

```diff
@@ -3482,13 +3483,16 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate', method='auto'
         'raise' throws an error, 'omit' performs the calculations ignoring nan
         values. Default is 'propagate'. Note that if the input contains nan
         'omit' delegates to mstats_basic.kendalltau(), which has a different
-        implementation.
+        implementation and currently only supports tau-a and tau-b variants.
     method: {'auto', 'asymptotic', 'exact'}, optional
         Defines which method is used to calculate the p-value [5]_.
         'asymptotic' uses a normal approximation valid for large samples.
         'exact' computes the exact p-value, but can only be used if no ties
         are present. 'auto' is the default and selects the appropriate
         method based on a trade-off between speed and accuracy.
+    variant: {'taua', 'taub', 'tauc'}, optional
+        Defines which variant of Kendall Tau is used. Different variants
+        use different normalizations, but all all proportional to the same
```

Comment:
Mention default. 

"all all" -> all

Reply:
It's better to write the type as `str, optional` and then give the default choice and options.

Reply:
The way I've written it seems to be consistent with other parts of scipy. I will, however, add the default choice.

Reply:
I agree that the set notation is used throughout scipy.stats.